apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -125,47 +126,49 @@ export default function ListBoxSearch({
   }
 
   return (
-    <StyledOutlinedInput
-      startAdornment={
-        <StyledInputAdornment position="start">
-          <Search size={dense ? 'small' : 'normal'} />
-        </StyledInputAdornment>
-      }
-      className="search"
-      sx={[
-        {
-          border: 'none',
-          borderRadius: 0,
-          backgroundColor: 'transparent',
-          '& fieldset': {
-            border: `1px solid ${theme.palette.divider}`,
-            borderWidth: '1px 0 1px 0',
-            borderRadius: 0,
-          },
-          '&:hover': {
+    searchEnabled !== false && (
```

Comment:
Maybe cleaner to move this check into the if statement on line 124 instead?
like this:
```
if (!visible || searchEnabled === false) {
  return null;
}
```